tests/waku_rln_relay/test_rln_group_manager_onchain.nim

```diff
@@ -168,18 +165,16 @@ proc setup(signer = true): Future[OnchainGroupManager] {.async.} =
   let web3 = await newWeb3(EthClient)
 
   let accounts = await web3.provider.eth_accounts()
-  web3.defaultAccount = accounts[1]
+  web3.defaultAccount = accounts[0]
```

Comment:
Shall we check the number of accounts is > 0?

```suggestion
  assert accounts.len > 0, "Not enough accounts retrieved"
```

Reply:
correct, the eth_accounts will always be > 0, since we're using a test rpc provider - ganache